app/containers/GamePage/bang.js

```diff
@@ -132,6 +164,27 @@ export default class Bang extends Component {
     });
   }
 
+  endTurn = () => {
+    const { database, rid, participants } = this.props;
+    const databaseRef = database.ref(`${rid}`);
+    const usersRef = database.ref(`${rid}/participants`);
+    databaseRef.once('value', (data) => {
+      const currentPlayer = data.val().currentPlayer;
+      if (currentPlayer + 1 > participants.length - 1) {
+        usersRef.child(participants[currentPlayer].key).update({ turn: false });
+        usersRef.child(participants[0].key).update({ turn: true });
+        database.ref(`${rid}`).update({ currentPlayer: 0 });
+        this.setState({ currentPlayer: 0 });
+      } else {
+        usersRef.child(participants[currentPlayer].key).update({ turn: false });
+        usersRef.child(participants[currentPlayer + 1].key).update({ turn: true });
+        this.setState({ currentPlayer: currentPlayer + 1 });
+        database.ref(`${rid}`).update({ currentPlayer: currentPlayer + 1 });
+      }
```

Comment:
const nextPlayer = (currentPlayer + 1) % participants.length; and use nextPlayer